ee/modules/140-user-authz/templates/multitenancy.yaml

```
@@ -0,0 +1,9 @@
+{{- if .Values.userAuthz.enableMultiTenancy }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: d8-multitenancy
```

```suggestion
  name: multitenancy-is-enabled
```
It is our namespace, so we don't need d8 prefix.